randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2143,6 +2143,12 @@ Extra - asset_id: collision_camera_032
           All of the following:
               Gravity Suit
               Simple IBJ or Use Spin Boost
+          All of the following:
+              # Ledge Warp (Advanced Movement) makes this viable https://www.youtube.com/watch?v=ND5zby3zaz8
+              Diagonal Bomb Jump (Hypermode) and Infinite Bomb Jump (Hypermode) and Knowledge (Advanced) and Lay Normal Bomb
+          All of the following:
+              # Ledge Warp is optional but makes attempts easier https://www.youtube.com/watch?v=AqlUANN4F_k
+              Cross Bomb and Movement (Advanced) and Can Slide
```

Comment:
Surely this has to be just "Slide" instead of the template.

Reply:
Oops I definitely forgot how the template was laid out. Yes this should be slide and lay cross bomb.